### PR TITLE
docs(EMPTY): fix broken link, updated description

### PR DESCRIPTION
**Pull request by nikagar4**
**Description:**

At some point we’d need to move `empty()` examples to `EMPTY` so I just moved it now,
since I already moved my codebase to `EMPTY` and there’s still no description in official docs

**Related issue (if exists):**

### CODE REVIEW

File: src/internal/observable/empty.ts [hunk 1]
```diff
@@ -57,7 +87,7 @@ export const EMPTY = new Observable<never>(subscriber => subscriber.complete());
  * the emission of the complete notification.
  * @return An "empty" Observable: emits only the complete
  * notification.
- * @deprecated Deprecated in favor of using {@link EMPTY} constant, or {@link scheduled} (e.g. `scheduled([], scheduler)`)
+ * @deprecated Deprecated in favor of using {@link index/EMPTY} constant, or {@link scheduled} (e.g. `scheduled([], scheduler)`)
```
Comment (nikagar4) [inline src/internal/observable/empty.ts]: If there's better way to refer to right link feel free to correct me.

Didn't have much time to dig into dgeni to fix this issue

Comment (benlesh) [reply src/internal/observable/empty.ts]: cc @JWO719

File: src/internal/observable/empty.ts [hunk 2]
```diff
@@ -4,6 +4,36 @@ import { SchedulerLike } from '../types';
 /**
  * The same Observable instance returned by any call to {@link empty} without a
  * `scheduler`. It is preferrable to use this over `empty()`.
+ *
+ * ## Examples
```
Comment (benlesh) [inline src/internal/observable/empty.ts]: I find both of these examples confusing... but I do want to thank you for pointing out that we need good examples of when to use `EMPTY`. You can remove these examples, and we'll get the fix to the link merged. (Unless @JWO719 can come up with a better fix for the links)

Comment (nikagar4) [reply src/internal/observable/empty.ts]: Agreed but couldn’t think of something small and clear, at least this got your attention.

Tried to come up with clear real life examples but when I wrote it I realized I introduced lots of other operators which confuses people who actually need to read these

File: src/internal/observable/empty.ts (hunk 1 above)
Comment (nikagar4) [reply src/internal/observable/empty.ts]: Looks like he’s not answering

**Comment by nikagar4** [comment]
> LGTM, thanks for the fix!

Always a pleasure :)

**Pull request merged**